torch/fx/experimental/proxy_tensor.py

```diff
@@ -382,6 +383,24 @@ def wrapped(*args):
 
     return wrapped
 
+class RetracingMode:
```

Comment:
I know this is not ideal... and I hate to save the interpreter in a global variable. 

but AOTAutograd wraps gm.forward with multiple layers of wrapper function, and by the time it reaches the make_fx function, there is no good way to access the binded interpreter anymore. 

Is there a better way to do this?

Reply:
hrmmm.... I think it's acceptable. Perhaps we should put this info on the ProxyTensorMode instead?

Reply:
I suppose you mean ProxyTorchDispatchMode... and I am not sure how to do that
We can chat offline if you think it's still doable that way.